backend/e4h-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -1647,4 +1655,352 @@ def process_update_incident_data_response(response, df, idx):
+                    df.at[index, 'Project Linking Status'] = "Not Attempted"
+                    continue
+
+                # handle create response
+                if create_resp.status_code in (200, 201):
+                    # keep validation status column untouched; write creation column
+                    created_id = None
+                    try:
+                        facilities = create_resp.json()
+                        if isinstance(facilities, list) and len(facilities) > 0:
+                            created_id = facilities[0].get("facility_id")
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)